.github/workflows/staging_deploy_real_bat.yml

```diff
@@ -0,0 +1,69 @@
+# This workflow will deploy in development environment the cycler controller module 
+
+name: Deployment of battery cycler controller to real environment
+
+on: workflow_call
+
+env:
+  ENV_CRED_TO_DEPLOY: ${{ secrets.ENV_CRED_TO_DEPLOY }}
+        
+jobs:
+  launch-real-bat:
+    name: Launch cycler controller in real environment
+    runs-on: [self-hosted, linux, ARM64, plc]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Test to get var and secrets
+        run: |
+          echo "GITHUB_REPOSITORY: $GITHUB_REPOSITORY"
+          echo "GITHUB_VAR: ${{ vars.CYCLER_STATION_ID }}"
+          echo "$ENV_CRED_TO_DEPLOY"
+      # - name: Set up Docker Buildx
```

Comment:
The image should not be downloaded to the host machine?

Reply:
The image is rebuilt in host machine to avoid long times during artifacts upload and download. Commented code removed.